Tribler/Main/vwxGUI/list_details.py

```diff
@@ -129,6 +129,9 @@ def __init__(self, parent):
 
         self.parent = parent
         self.torrent = Torrent('0', '0', '', 0, 0, 0, 0, 0, None)
+        self.torrent.torrent_db = self.guiutility.channelsearch_manager.torrent_db
```

Comment:
I think I added this because some other code was using `torrent.torrent_db` and `torrent.channelcast_db` as well, but I don't remember which part of code any more...